lib/Dancer2/Core/App.pm

```diff
@@ -833,6 +833,41 @@ sub log {
     $logger->$level(@_);
 }
 
+sub send_as {
+    my $self = shift;
+    my ( $type, $data, $options ) = @_;
+    $options ||= {};
+
+    # allow lower cased serializer names
+    my $prefix = 'Dancer2::Serializer::';
+    my $serializer_class;
+    for ( uc $type, $type ) {
+        if ( Class::Load::try_load_class( "$prefix$_" ) ) {
```

Comment:
Can we replace [Class::Load](https://metacpan.org/pod/Class::Load) with [Module::Runtime](https://metacpan.org/pod/Module::Runtime)?

Ooo, can we benchmark them against [Class::Load::XS](https://metacpan.org/pod/Class::Load::XS) to make sure [Module::Runtime](https://metacpan.org/pod/Module::Runtime) is also faster?